package.json

```diff
@@ -60,13 +60,14 @@
     "babel-loader": "8.2.2",
     "babel-preset-env": "7.0.0-beta.3",
     "chai": "4.3.0",
-    "closure-webpack-plugin": "2.5.0",
+    "closure-webpack-plugin": "^2.5.0",
```

Comment:
No action needed but just a heads up that renovate will pin these dependencies to a specific version (in a subsequent PR) instead of using the compatible caret range here.